src/components/o-modal/_preview.html

```diff
@@ -37,36 +42,106 @@ <h2>Prämie berechnen</h2>
 </script>
 
 <style>
-  .axa-modal-preview axa-container {
-    margin-top: 70px;
-  }
 
-  .axa-modal-preview h2 {
-    font-family: PublicoHeadline;
-    font-weight: bold;
-    color: #333333;
-    font-size: 36px;
-    margin-bottom: 50px;
-  }
+    .axa-modal-preview .padded-text-col {
+      margin-top: 10px;
+      padding-left: 10px;
+      font-size: 16px;
 
-  .axa-modal-preview p {
-    font-size: 20px;
-    line-height: 1.5;
-    color: #333333;
-  }
+      margin-bottom: 15px;
+    }
 
-  .axa-modal-preview .huge {
-    height: 70px;
-    width: 70px;
-    color: #00008f;
-  }
+    .axa-modal-preview axa-container {
+      margin-top: 20px;
 
-  .axa-modal-preview axa-icon[icon='check_circle_outline'] {
-    fill: green;
-    color: white;
-  }
+      padding-left: 0%;
+      padding-right: 0%;
+    }
 
-  .axa-modal-preview axa-col {
-    padding: 10px;
-  }
+    @media (min-width: 768px) {
+        .axa-modal-preview axa-container {
+          padding-left: 5%;
+          padding-right: 5%;
+        }
+    }
+
+    @media (min-width: 1200px) {
+        .axa-modal-preview axa-container {
+          padding-left: 10%;
+          padding-right: 10%;
+        }
+    }
+
+    .axa-modal-preview axa-container .m-col .m-button {
+        width: 100%;
+    }
+
+    .axa-modal-preview .m-modal__close-button {
+        float: none;
+        position: absolute;
+        right: 20px;
+
+    }
+
+    .axa-modal-preview .m-modal__content {
+        position: relative;
+
+    }
+
+    .axa-modal-preview h2 {
```

Comment:
no globals, right?

Reply:
yes right but this is not global. is scoped in axa-preview. And this is copy paste from previous :)